rust/src/database/verkle/variants/simple/node.rs

```diff
@@ -34,21 +37,21 @@ impl Node {
 
     /// Stores the value for the given key.
```

Comment:
```suggestion
    /// Stores the given updates.
```

Or "update batch", or "key value batch", depending on what we settle on naming wise. Also below.